tests/e2e/framework/kubernetes.go

```diff
@@ -316,7 +318,7 @@ func (k *KubeInfo) generateSidecarInjector(src, dst string) error {
 		content = updateIstioYaml("sidecar_injector", *pilotHub, *pilotTag, content)
```

Comment:
@sebastienvas I was not sure if the sidecar_injector should use the pilot tag or the proxy tag.